src/styles/variables.scss

```
@@ -0,0 +1,18 @@
+$color-blue: #00ACDC;
+$color-white: #fff;
+$color-dark: #060B35;
+$color-grey: #616070;
+$color-hover-text: #34568b;
+$color-blue: #00ACDC;
+$color-basic: #000;
+$coklor-basic-link: #00acdc;
+$color-back: #FFF;
```

Why are you creating multiple variables with the same value?
```suggestion
$color-blue: #00ACDC;
$color-white: #fff;
$color-dark: #060B35;
$color-grey: #616070;
$color-hover-text: #34568b;
$color-basic: #000;
```